examples/inverse/plot_use_sourcemorph.py

```diff
@@ -0,0 +1,99 @@
+"""
+================================
+Demonstrate usage of SourceMorph
+================================
+
+This example demonstrates how to morph an individual subject source estimate to
+a common reference space. It will be demonstrated using the SourceMorp class.
+The example uses parts of the MNE example
+:ref:`sphx_glr_auto_examples_inverse_plot_lcmv_beamformer_volume.py` and
+:ref:`sphx_glr_auto_examples_inverse_plot_lcmv_beamformer.py`.
+The respective result will be morphed based on an affine transformation and a
+nonlinear morph, estimated based on respective transformation from the
+subject's anatomical T1 (brain) to fsaverage T1 (brain) in VolSourceEstimate
+case and using an affine transform in the SourceEstimate or
+VectorSourceEstimate case. Afterwards the transformation will be applied to the
+beamformer result. The result will be a plot showing the morphed result
+overlaying the fsaverage T1. Uncomment at the respective location to plot the
+result of the surface morph.
+
+"""
+# Author: Tommy Clausner <tommy.clausner@gmail.com>
+#
+# License: BSD (3-clause)
+
+import matplotlib.pylab as plt
+import nibabel as nib
+import numpy as np
+from mne import read_evokeds, SourceMorph
+from mne.datasets import sample
+from mne.minimum_norm import apply_inverse, read_inverse_operator
+from nilearn.plotting import plot_anat
+
+print(__doc__)
+
+###############################################################################
+# Setup paths
+sample_dir = sample.data_path() + '/MEG/sample'
+subjects_dir = sample.data_path() + '/subjects'
+
+fname_evoked = sample_dir + '/sample_audvis-ave.fif'
+fname_inv = sample_dir + '/sample_audvis-meg-vol-7-meg-inv.fif'
+
+fname_t1_fsaverage = subjects_dir + '/fsaverage/mri/brain.mgz'
+
+###############################################################################
+# Compute example data. For reference see
+# :ref:`<sphx_glr_auto_examples_inverse_plot_compute_mne_inverse_volume.py>`
+
+# Load data
+evoked = read_evokeds(fname_evoked, condition=0, baseline=(None, 0))
+inverse_operator = read_inverse_operator(fname_inv)
+
+# Apply inverse operator
+stc = apply_inverse(evoked, inverse_operator, 1.0 / 3.0 ** 2, "dSPM")
+
+# To save memory
+stc.crop(0.087, 0.087)
+
+###############################################################################
+# Morph VolSourceEstimate
+
+# Initialize SourceMorph for VolSourceEstimate
+source_morph = SourceMorph(inverse_operator['src'],
+                           subject_from='sample',
+                           subject_to='fsaverage',
+                           subjects_dir=subjects_dir,
+                           grid_spacing=(5., 5., 5.))
+
+# Save and load SourceMorph
+# source_morph.save('vol')
+# source_morph = mne.read_source_morph('vol-morph.h5')
+
+# Morph data
+np.abs(stc.data, out=stc.data)  # for plotting
```

Comment:
this is pretty exotic for a new user. I would provide a one line reason why we don't do `stc.data = np.abs(stc.data)`

Reply:
actually, is it equivalent to do `abs` then morph vs. doing the morph and then taking the `abs`. That is to say, is the morphing linear?

Reply:
it's a nonlinear morph... it has to be like it is